packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/tvl-breakdown/_components/tables/canonically-bridged-table.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import {
+  type ExpandedState,
+  getCoreRowModel,
+  getExpandedRowModel,
+} from '@tanstack/react-table'
+import { useState } from 'react'
+import { TokenTable } from '~/app/_components/table/token-breakdown-table'
+import { useTable } from '~/hooks/use-table'
+import { type ExtendedProjectTvlBreakdown } from '../../_utils/assign-token-meta-to-breakdown'
+import { canonicallyBridgedColumns } from './columns/canonically-bridged-columns'
+import { TableSum } from './table-sum'
+
+export type CanonicallyBridgedTokenEntry =
+  ExtendedProjectTvlBreakdown['breakdown']['canonical'][number]
+
+interface Props {
+  tokens: CanonicallyBridgedTokenEntry[]
+}
+
+export function CanonicallyBridgedTable(props: Props) {
+  const sum = props.tokens.reduce((acc, token) => {
+    return acc + Number(token.usdValue)
+  }, 0)
```

Comment:
it's used in every table file, let's simply extract it to utils

Reply:
https://github.com/l2beat/l2beat/pull/4855/commits/ea18d09d1b58e9512fddd01181d49a30b7d82e01